content/create-new-collection.en.md

```diff
@@ -0,0 +1,160 @@
+---
+title: "Creating a collection"
+weight: 2
+---
+
+# Creating a collection
+
+## Define the necessary metadata
+
+First of all, it is strongly recommended to define the scope and metadata of the collection:
+- Concise description of the collection topic (examples: `Largest global social media`, `Most used social media in France`, `Dating apps`, `Platforms providing services to businesses`…)
+- Collection name (3 words maximum, examples: `Platform Governance Archive`, `France Élections`, `Dating`, `P2B Compliance Assessment`…)
+- Collection ID (examples: `pga`, `France-elections`, `dating`, `p2b-compliance`…)
+- Terms language (examples: `English`, `French`, `English`, `All EU languages`…)
+- Service jurisdiction (examples: `EU`, `France`, `EEA`, `EU`…)
+- Collection maintainer entity (name, logo, url)
+
+## Create repositories
+
+### Collection definition
```

Comment:
At this stage, we are still using the term “declarations” throughout the documentation, I believe it is better to be consistent and make a mass change when / if we decide to replace “declarations” by “definitions”.

```suggestion
### Declarations
```